packages/apps/graph/generated-schema.graphql

```diff
@@ -196,9 +237,6 @@ type PageInfo {
 scalar PositiveFloat
 
 type Query {
-  """Retrieve an account by its name and fungible, such as coin."""
-  account(accountName: String!, fungibleName: String!): FungibleAccount
-
```

Comment:
@Ghislain89 this will affect the integration tests. Should I also change them accordingly?